### PR TITLE
Update URL of author inside the author configuration

### PR DESCRIPTION
**Pull request by PrestaEdit**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.x
| Description?  | As author URL is excluded from PrestaShop Addons rules, mention it.

### CODE REVIEW

**Review by Progi1984 — COMMENTED** [review]
I'm not a big fan of this PR.

Why ? We are currently in split of Company and Project. So we try to remove specific informations about Addons.

**Comment by PrestaEdit** [comment]
I know. But I need to make this PR in order to remember be this situation.
We could maybe remove the "theme_key" and add this specific part (theme_key and add notice about URL) on the "Distribute on Addons".

**Comment by matks** [comment]
The "Distribute on Addons" section is supposed to move in another website in the future, so that knowledge is not lost. It will be somewhere near Addons naturally.

**Comment by PrestaEdit** [comment]
_Supposed, future, maybe, ..._

So, it's not done, yet.
So, we remove this now or we could add this kind of informations as themes developpers facing this problem on market validation right now, not in the future. :)

File: themes/getting-started/theme-yml.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Users will be able to choose the layout for each page from the theme's settings
   author:
     name: "John Doe"
     email: "pub@prestashop.com"
-    url: "https://www.prestashop.com"
+    url: "https://www.prestashop.com" # Avoided by PrestaShop Addons, don't add it if you want to distribute the theme on Addons Platform
```
Comment (PierreRambaud) [inline themes/getting-started/theme-yml.md]:
```suggestion
    url: "https://www.prestashop-project.org/"
```

**Comment by PierreRambaud** [comment]
Let's change the URL by `https://www.prestashop-project.org/` and we don't care about what Addons wants or not. An author should be defined by a name, an email and an URL.

File: themes/getting-started/theme-yml.md (hunk 1 above)
Comment (matks) [reply themes/getting-started/theme-yml.md]: Good idea IMO 👍

Comment (eternoendless) [reply themes/getting-started/theme-yml.md]: Even better:
```suggestion
    url: "https://www.your-url.com" # Be aware that some marketplaces don't allow this parameter
```

**Comment by PrestaEdit** [comment]
Hi all,

I've removed the notice about Addons, and so the title of the PR but note sure that it's really needed, so, to merge this as it.

**Review by Progi1984 — APPROVED** [review]

File: themes/getting-started/theme-yml.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Users will be able to choose the layout for each page from the theme's settings
   author:
     name: "John Doe"
     email: "pub@prestashop.com"
-    url: "https://www.prestashop.com"
+    url: "https://www.your-url.com"
```
Comment (atomiix) [inline themes/getting-started/theme-yml.md]: Could you add the comment as stated by @eternoendless in [this comment](https://github.com/PrestaShop/docs/pull/1137#discussion_r727312770)?

**Review by atomiix — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
thank you @PrestaEdit 👍

**Comment by eternoendless** [comment]
This will need to be backported into 8.x

**Comment by matks** [comment]
> This will need to be backported into 8.x

`8.x` branch content is updated

**Pull request merged**